suites/quincy/rbd/tier-2_rbd_regression.yaml

```diff
@@ -106,3 +106,10 @@ tests:
       module: rbd_trash.py
       name: Disable Trash and Delete images and check if they are not moved to trash automatically
       polarion-id: CEPH-83573296
+
+  - test:
+      desc: verify image doesnot exist after Moving images to trash and apply force purge to images
+      destroy-cluster: false
+      module: rbd_trash_purge_delete_trash_image.py
```

Comment:
If all trash-related operation in one module, can we make the module name shorter?

Reply:
done.